Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py

```diff
@@ -726,3 +726,51 @@ def test_get_modified_remote_data_command(requests_mock):
     response = get_modified_remote_data_command(client, args)
 
     assert response.modified_incident_ids == ['1', '2']
+
+
+def test_get_contributing_event_command(requests_mock):
+    from CortexXDRIR import get_contributing_event_command, Client
+
+    contributing_events = load_test_data('./test_data/contributing_events.json')
+    requests_mock.post(f'{XDR_URL}/public_api/v1/alerts/get_correlation_alert_data/', json=contributing_events)
+
+    client = Client(
+        base_url=f'{XDR_URL}/public_api/v1', headers={}
+    )
+    args = {
+        "alert_ids": "[1111]",
+    }
+
+    response = get_contributing_event_command(client, args)
+
+    assert response.outputs[0].get('alertID') == args.get('alert_ids').strip('[]')
```

Comment:
@dansterenson I left it that way because using argToList convert it to an int and but it is a string.